api/internal/handler/resources/resources.go

```diff
@@ -91,12 +91,19 @@ func (h *Handler) setupClient(cfg config.Config) {
 func (h *Handler) ProxyResource(c *gin.Context, _ interface{}) handler.Response {
 	resp, err := h.proxy(c)
 	if err != nil {
-		return *err
+		return handler.Response{
+			Success: false,
+			ErrMsg:  err.Error(),
+		}
```

Comment:
We'd better not change it for now, maybe it's better to address the code style issue in a later PR.